build.gradle

```diff
@@ -1,5 +1,8 @@
 import edu.wpi.first.gradlerio.GradleRIOPlugin
 import groovy.json.JsonSlurper
+import java.nio.file.Files
+import java.nio.file.NotLinkException
+import java.io.FileFilter
```

Comment:
spotless should remove these

Reply:
I don't think it possibly could, it doesn't know which imports are actually used. But good catch